simpleprophet/simpleprophet/data.py

```diff
@@ -60,7 +60,7 @@ def get_kpi_data(bq_client, types=tuple(KPI_QUERIES.keys())):
             if q == k.lower():
                 q = k
                 break
-        else:
+        if q not in KPI_QUERIES.keys():
```

Comment:
The `else` here is intentional. It should be executed only if the `for` loop runs to completion without any of the cases `break`ing out. It's a lesser-used feature, so I can add a comment here to clarify.

Have this caused any functional problems?

Reply:
I think it means that the types argument has to be lower case or we raise an error?  Or am I misunderstanding?

Reply:
The intention was to accept any casing for the input and this code would normalize the value of `q` to match the expected casing. But I think you're correct that this now only supports `q` being lowercase. Will fix.

Reply:
Actually, I like your solution. Let's merge it.